huaweicloud/services/ddm/data_source_huaweicloud_ddm_accounts.go

```diff
@@ -0,0 +1,212 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product DDM
+// ---------------------------------------------------------------
+
+package ddm
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func DataSourceDdmAccounts() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: resourceDdmAccountsRead,
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the ID of DDM instance.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Specifies the name of the DDM account.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Specifies the status of the DDM account.`,
+			},
+			"accounts": {
+				Type:        schema.TypeList,
+				Elem:        AccountsAccountSchema(),
+				Computed:    true,
+				Description: `Indicates the list of DDM account.`,
+			},
+		},
+	}
+}
+
+func AccountsAccountSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the name of the DDM account.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the status of the DDM account.`,
+			},
+			"permissions": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Computed:    true,
+				Description: `Indicates the basic permissions of the DDM account.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the description of the DDM account.`,
+			},
+			"schemas": {
+				Type:        schema.TypeList,
+				Elem:        AccountsAccountSchemaSchema(),
+				Computed:    true,
+				Description: `Specifies the schemas that associated with the account.`,
```

Comment:
Replace `Specifies` with `Indicates`, check the other parts.

Reply:
done